src/Framework/MSBuildEventSource.cs

```diff
@@ -476,13 +476,13 @@ public void WriteLinesToFileUpToDateStop(string fileItemSpec, bool wasUpToDate)
         }
 
         [Event(62, Keywords = Keywords.All)]
-        public void SdkResolverServiceInitializeStart()
+        public void SdkResolverLoadAllResolversStart()
         {
             WriteEvent(62);
         }
 
         [Event(63, Keywords = Keywords.All)]
-        public void SdkResolverServiceInitializeStop(int resolverCount)
+        public void SdkResolverLoadAllResolversStop(int resolverCount)
```

Comment:
@jeffkl, I believe that you added some of these events. Can you please confirm that renaming is ok, i.e. nothing on your side depends on the names. Thank you!

Reply:
I am not personally aware of any dependency on the names of these events

Reply:
My read of https://learn.microsoft.com/en-us/dotnet/core/diagnostics/eventsource-instrumentation#best-practices rule 7 is that renaming should imply giving these new numbers, but it's not that clear to me.